dimagi/commcare-hq
# Case search screen subtitle/description text

## Product Description
Adds "Search Screen Subtitle" to case search screen options and shows this text on case search query screen.

In app manager:
<img width="500" alt="Screen Shot 2022-11-16 at 3 20 45 PM" src="https://user-images.githubusercontent.com/100609770/202315849-cfa43feb-8aae-499c-a450-53a407b6e407.png">


In case search:
![Screen Shot 2022-11-14 at 10 08 27 AM](https://user-images.githubusercontent.com/100609770/201734329-a222f36b-61ba-4312-bafe-a67b50abe4fa.png)


## Technical Summary
[USH-2366](https://dimagi-dev.atlassian.net/browse/USH-2366)

- Adds `description` to app manager screen with a new `textarea_trans` field in the style of `input_trans`. 
- Saves to CaseSearch model as a LabelProperty. 
- Uses app_strings and id_strings to make text translatable, with additions to bulk upload/download modules to allow bulk translation.
- Sends `<description>` field as part of RemoteRequestQuery XML to be read by Formplayer. 
- Renders with markdown on case search screen using `markdown-it` and `DOMPurify`.

Related changes to commcare-core in https://github.com/dimagi/commcare-core/pull/1164 and formplayer in https://github.com/dimagi/formplayer/pull/1267

## Feature Flag
SYNC_SEARCH_CASE_CLAIM

## Safety Assurance

### Safety story
Tested workflows locally including translations, as well as using "dirty" strings to check for proper sanitizing. Ran automated tests known to touch these areas and added description text to each.

### Automated test coverage
Added coverage in case claim bulk translation tests.

### QA Plan
Will send to QA to ensure proper entry, translation, and display of description text.
[QA-4676](https://dimagi-dev.atlassian.net/browse/QA-4676?atlOrigin=eyJpIjoiODljZmQ2NWM1NzhlNDIwNjg4ODQ5MmMxYjc2NTBmZjYiLCJwIjoiaiJ9)

### Rollback instructions




### Labels & Review
